Update CONTRIBUTING.md (#627)
open-telemetry/opentelemetry-proto: CONTRIBUTING.md

# Contributing

## Introduction

Welcome, and thank you for your interest in contributing to the OpenTelemetry Protocol (OTLP) Specification! Your contributions—big or small—are invaluable in shaping and improving this essential part of the OpenTelemetry ecosystem.

> Whether you are fixing a small issue, updating documentation, or introducing a major improvement, we appreciate your efforts. If you’re new to the project, don’t hesitate to ask questions and seek guidance from the community—we are here to support you!

Reviewer: ```suggestion
Whether you are fixing a small issue, updating documentation, or introducing a major improvement, we appreciate your efforts.
If you’re new to the project, don’t hesitate to ask questions and seek guidance from the community.
We are here to support you!
```

# Contributing

## Introduction

> Welcome, and thank you for your interest in contributing to the OpenTelemetry Protocol (OTLP) Specification! Your contributions—big or small—are invaluable in shaping and improving this essential part of the OpenTelemetry ecosystem.

Reviewer 2: ```suggestion
Welcome, and thank you for your interest in contributing to the OpenTelemetry Protocol (OTLP) Specification! Your contributions — big or small — are invaluable in shaping and improving this essential part of the OpenTelemetry ecosystem.
```

Whether you are fixing a small issue, updating documentation, or introducing a major improvement, we appreciate your efforts.

> If you’re new to the project, don’t hesitate to ask questions and seek guidance from the community.

Reviewer 2: ```suggestion
If you're new to the project, don't hesitate to ask questions and seek guidance from the community.
```

After making any changes to .proto files make sure to generate all
implementation by running `make gen-all`.

## Further Help

> If you have any questions or need assistance while contributing, feel free to reach out to the [otel-specification](https://cloud-native.slack.com/archives/C01N7PP1THC) slack channel. 

Reviewer: ```suggestion
If you have any questions or need assistance while contributing, feel free to reach out to the [`otel-specification`](https://cloud-native.slack.com/archives/C01N7PP1THC) Slack channel. 
```

> View meeting notes of previous SIG calls too [here](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo/edit?tab=t.0#heading=h.w5ybybd67v4h) to stay up to date.

Reviewer: We are rotating the docs from time to time. Maybe better giving a hyperlink to https://github.com/open-telemetry/opentelemetry-specification which is stable?

Author: > We are rotating the docs from time to time. Maybe better giving a hyperlink to https://github.com/open-telemetry/opentelemetry-specification which is stable?

Alright

After making any changes to .proto files make sure to generate all
implementation by running `make gen-all`.

## Further Help

If you have any questions or need assistance while contributing,

> feel free to reach out to the [`otel-specification`](https://cloud-native.slack.com/archives/C01N7PP1THC) Slack channel. 

Reviewer 3: ```suggestion
feel free to reach out to the [`#otel-specification`](https://cloud-native.slack.com/archives/C01N7PP1THC) Slack channel. 
```

> View meeting notes of previous SIG calls too [here](https://github.com/open-telemetry/opentelemetry-specification) to stay up to date.

Reviewer 3: This link doesn't have meeting notes. Meeting notes are in this [google doc](https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s/edit?gid=0#gid=0) as [documented here](https://github.com/open-telemetry/community/?tab=readme-ov-file#governing-bodies).

> View meeting notes of previous SIG calls in this [google doc](https://docs.google.com/spreadsheets/d/1SYKfjYhZdm2Wh2Cl6KVQalKg_m4NhTPZqq-8SzEVO6s/edit?gid=0#gid=0) as stated [here](https://github.com/open-telemetry/community/?tab=readme-ov-file#governing-bodies) to stay up to date.

Reviewer 4: Please keep the link to spec repo too.
https://github.com/open-telemetry/opentelemetry-specification?tab=readme-ov-file#questions

Author: Done @cijothomas

After making any changes to .proto files make sure to generate all
implementation by running `make gen-all`.

## Further Help

If you have any questions or need assistance while contributing, feel free to reach out to the [`#otel-specification`](https://cloud-native.slack.com/archives/C01N7PP1THC) Slack channel.  

> 
> 

Reviewer 2: ```suggestion

```
